### PR TITLE
fix(account): update stored procedures to be more replication friendly

### PR DESCRIPTION
**Pull request by vbudhram**
This PR updates our stored procedures to be more replication friendly and is targeted against train-121.

- [x] verifyTokensWithMethod_2
- [x] verifyTokenCode_1

### CODE REVIEW

File: lib/db/schema/patch-086-087.sql [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+)
+BEGIN
+  DECLARE EXIT HANDLER FOR SQLEXCEPTION
+  BEGIN
+    ROLLBACK;
+    RESIGNAL;
+  END;
+
+    -- Update session verification methods
+    UPDATE `sessionTokens` SET verificationMethod = verificationMethodArg, verifiedAt = verifiedAtArg
```
Comment (vbudhram) [inline lib/db/schema/patch-086-087.sql]: I significantly tweaked this procedure to only update the` verificationMethod` on the session table. All previous logic was pulled into the `db.verifyTokensWithMethod` procedure.

File: lib/db/schema/patch-086-087.sql [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      SIGNAL SQLSTATE '45000' SET MYSQL_ERRNO = 2101, MESSAGE_TEXT = 'Expired token verification code.';
+    END IF;
+  END IF;
+
+  UPDATE securityEvents
+  SET verified = true
+  WHERE tokenVerificationId = @tokenVerificationId
+  AND uid = uidArg;
+
+  DELETE FROM unverifiedTokens
```
Comment (vbudhram) [inline lib/db/schema/patch-086-087.sql]: I simplified this by removing the transaction portion, which will return the `affectedRows` from the `DELETE`. I don't remember the original logic why I put this in a transaction but think it is ok to remove.

**Comment by vbudhram** [comment]
@mozilla/fxa-devs @jrgm @rfk Thoughts on approach?

File: lib/db/schema/patch-086-087.sql (hunk 2 above)
Comment (jrgm) [reply lib/db/schema/patch-086-087.sql]: What a pain. It does kinda wanna be a transaction (I logged it, I did it), but...

**Pull request merged**